go/enclave/components/batch_executor.go

```diff
@@ -72,14 +72,14 @@ func (executor *batchExecutor) payL1Fees(stateDB *state.StateDB, context *BatchE
 	for _, tx := range context.Transactions {
 		sender, err := core.GetAuthenticatedSender(context.ChainConfig.ChainID.Int64(), tx)
 		if err != nil {
-			executor.logger.Warn("Unable to extract sender for tx", log.TxKey, tx.Hash())
+			executor.logger.Debug("Unable to extract sender for tx", log.TxKey, tx.Hash(), log.ErrKey, err)
```

Comment:
Those two here are warnings as they should never happen. Being unable to extract the sender means the tx here is weirdly signed and should've been cut way earlier

Reply:
gotcha.